src/disco/components/Addon.js

```diff
@@ -116,43 +112,27 @@ export class AddonBase extends React.Component {
   }
 
   getThemeImage() {
-    const { getBrowserThemeData, i18n, name, previewURL } = this.props;
+    const { i18n, name, previewURL } = this.props;
     if (this.props.type === ADDON_TYPE_THEME) {
       /* eslint-disable jsx-a11y/href-no-hash, jsx-a11y/anchor-is-valid */
       return (
-        <HoverIntent
-          onHoverIntent={this.previewTheme}
-          onHoverIntentEnd={this.resetThemePreview}
+        <a
+          href="#"
+          className="theme-image"
         >
-          <a
-            href="#"
-            className="theme-image"
-            data-browsertheme={getBrowserThemeData()}
-            onBlur={this.resetThemePreview}
-            onClick={this.installTheme}
```

Comment:
The old version of the `<a>` tag had an `onClick` property which was assigned to the `this.installTheme` function. Don't we still need that?

Reply:
i see what you mean but i am looking at about:addons and i don't see a click installing a theme...do you? let me know if i am missing something

the only install i see happening is if they toggle here: 
![Alt text](https://monosnap.com/image/lVheuMJOSoDtZKw5rGqOKeIsHpz6Xf.png)

which it looks like is implemented with the InstallButton component (below)

Reply:
As mentioned above I think clicking the image resulted in an install too.

Reply:
> As mentioned above I think clicking the image resulted in an install too.

Hmm it doesn't appear to currently work that way.

Reply:
ok i'll add this code back in : )